### PR TITLE
Add model-centric tutorial

### PR DESCRIPTION
**Pull request by jmaunon**
## Description
This MR will include model-centric tutorial (see https://github.com/OpenMined/PySyft/blob/master/examples/tutorials/model-centric-fl/Part%2001%20-%20Create%20Plan.ipynb) in our repository, passing through our integration test

- [x] Part 01
- [x] Part 02

## Affected Dependencies
None
## How has this been tested?
I have included a test (actually, it is here where I found some issues, but all of them fixed :) )

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by cereallarceny** [comment]
@vvmnnnkv please merge this when you're comfortable with the changes. We're just moving the model-centric tutorials from the PySyft repo to the PyGrid repo. Steps for you and your team after this PR is merged:
- [ ] Remove the tutorials from PySyft
- [ ] Change the links to the tutorials in syft.js
- [ ] Change the links to the tutorials in KotlinSyft
- [ ] Change the links to the tutorials in SwiftSyft

**Review by vvmnnnkv — APPROVED** [review]
Finally these notebooks are included in the test suite ❤️

**Pull request merged**
